org/unit.go

```diff
@@ -38,6 +39,9 @@ const (
 	UnitHour         Unit = `h`
 	UnitMinute       Unit = `min`
 	UnitPiece        Unit = `piece`
+	UnitEach         Unit = `each`
```

Comment:
"Each" is indeed weird. I used simply because that's the name that appears in the UNECE table and is short:

![image](https://github.com/invopop/gobl/assets/856/fd502928-1e06-439e-a513-f7b659564385)

But, on a second thought, in the MX catalogue, they call it "Elemento" and I've notice that "Item" is not used for anything else in UNECE. "Item" is also short and, although a bit too generic, definitely less strange than "Each". 

I was also thinking that perhaps we could make "Service" and "Activity" shorter: `serv` and `act`. WDYT?

Reply:
Okay, I'm included to agree that `item` (a noun) has more meaning than `each` (a determiner or pronoun). I was going to suggest that "each" could be understood for both services and products, but then the description says "item", which rules that idea out.

Think about this... it would be nice to have the official descriptions there too. Might be worth a quick copy and paste session there.

I think the `service` and `activity` names will be used so infrequently that we can keep the long versions for clarity.

Reply:
Feedback address in 04d35d6